templates/components/loaders/python/web/loader.py

```diff
@@ -1,13 +1,31 @@
 import os
+import json
+from pydantic import BaseModel, Field
 from llama_index.readers.web import WholeSiteReader
 
 
-def get_documents():
-    # Initialize the scraper with a prefix URL and maximum depth
-    scraper = WholeSiteReader(
-        prefix=os.environ.get("URL_PREFIX"), max_depth=int(os.environ.get("MAX_DEPTH"))
-    )
-    # Start scraping from a base URL
-    documents = scraper.load_data(base_url=os.environ.get("BASE_URL"))
+class WebLoaderConfig(BaseModel):
+    base_url: str
+    prefix: str
+    max_depth: int = Field(default=1, ge=0)
+
+
+def load_configs():
+    with open("config/loaders.json") as f:
+        configs = json.load(f)
+    web_config = configs.get("web", None)
+    if web_config is None:
+        raise ValueError("No web config found in loaders.json")
+    return [WebLoaderConfig(**config) for config in web_config]
 
+
+def get_documents():
+    web_config = load_configs()
```

Comment:
ok, i'll merge this now. in the next PR the naming should be more consistent, like:
```suggestion
    config = load_config()
```
or
```suggestion
    web_config = load_web_config()
```